src/Configuration/Configuration.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BackwardCompatibility\Configuration;
+
+use Roave\BackwardCompatibility\Baseline;
+
+/** @psalm-immutable */
```

Comment:
Possibly `@internal` for now? 🤔

Reply:
@Ocramius psalm isn't that happy due to the different top-level namespace for tests:

```sh
ERROR: InternalProperty - test/unit/Configuration/ConfigurationTest.php:18:47 - Roave\BackwardCompatibility\Configuration\Configuration::$baseline is internal to Roave but called from RoaveTest\BackwardCompatibility\Configuration\ConfigurationTest (see https://psalm.dev/176)
        self::assertEquals(Baseline::empty(), $config->baseline);


ERROR: InternalProperty - test/unit/Configuration/ConfigurationTest.php:19:26 - Roave\BackwardCompatibility\Configuration\Configuration::$filename is internal to Roave but called from RoaveTest\BackwardCompatibility\Configuration\ConfigurationTest (see https://psalm.dev/176)
        self::assertNull($config->filename);
```

Should we keep `@internal` and baseline these or should we remove `@internal`?

Reply:
I've removed for now.